views/settings.php

```diff
@@ -8,14 +8,36 @@
 use WP101\Migrate as Migrate;
 use WP101\TemplateTags as TemplateTags;
 
+$api_key = TemplateTags\get_api_key();
+
+/*
+ * Mask the API key, showing only the first four characters.
+ *
+ * The rest should be replaced with "&#9679;", a Unicode black circle.
+ *
+ * Masked keys will look something like: "ABCD●●●●●●●●●●●●●●●●●●●●●●●●●●●●".
+ */
+$masked = str_pad(
+	substr( $api_key, 0, 4 ),
+	4 + 7 * ( strlen( $api_key ) - 4 ),
+	'&#9679;',
+	STR_PAD_RIGHT
+);
+
 ?>
 
 <div class="wrap wp101-settings">
 	<h1><?php echo esc_html( _x( 'WP101 Settings', 'settings page title', 'wp101' ) ); ?></h1>
 
 	<?php settings_errors(); ?>
 
+	<h2 id="api-key"><?php echo esc_html( _x( 'WP101Plugin.com API Key', 'settings section heading', 'wp101' ) ); ?></h2>
```

Comment:
`esc_html_x` exists.

Reply:
That's the one I always seem to forget 🤦‍♂️